src/NServiceBus.Metrics.ServiceControl/ReportingFeature.cs

```diff
@@ -271,6 +271,11 @@ async Task Sender(byte[] body)
                     {
                         await dispatcher.Dispatch(new TransportOperations(operation), new TransportTransaction(), new ContextBag())
                             .ConfigureAwait(false);
+
```

Comment:
Would it help to measure dispatch time when debug is enabled?

Reply:
might be interesting. If this is a thread starvation issue as I currently suspect, this might also be visible by measuring the time it takes.

Reply:
Good inspiration 😉https://github.com/Azure/azure-sdk-for-net/pull/11266

Reply:
We decide to not do this because we know now that the contention happens in the extension that writes do the buffer